addons/stock/models/stock_production_lot.py

```diff
@@ -74,6 +78,18 @@ def _compute_display_complete(self):
         for prod_lot in self:
             prod_lot.display_complete = prod_lot.id or self._context.get('display_complete')
 
+    def _compute_delivery_ids(self):
+        delivery_ids_by_lot = self._find_delivery_ids_by_lot(self)
+        for lot in self:
+            lot.delivery_ids = self.env['stock.picking'].browse(delivery_ids_by_lot[lot.id])
+            lot.delivery_count = len(lot.delivery_ids)
+            # If lot is serial, keep track of the latest delivery's partner
+            if lot.product_id.tracking == 'serial' and lot.delivery_count > 0:
```

Comment:
Probably should be the same for lot if the whole pack is for one customer. To discuss with mgm